applications.json

```diff
@@ -1233,20 +1247,6 @@
                 "swift"
             ]
         },
-        {
-            "short_description": "macOS application to comfortably browse and search through your Messages.app history. ",
-            "categories": [
-                "git"
-            ],
-            "repo_url": "https://github.com/glaurent/MessagesHistoryBrowser",
-            "title": "MessagesHistoryBrowser",
-            "icon_url": "",
-            "screenshots": [],
-            "official_site": "",
-            "languages": [
-                "swift"
-            ]
-        },
```

Comment:
@Amorymeltzer Thank you! 👍 
You don't need to move all json object anywhere, you can just change "categories" value, and CI will regenerate readme file and put app into correct place. Could you minimise this changes?